src/internal/hooks/use-unique-id/index.ts

```diff
@@ -3,7 +3,7 @@
 import React, { useRef } from 'react';
 
 let counter = 0;
-const useIdFallback = () => {
+export const useIdFallback = () => {
```

Comment:
If we are going to use it this way, we should rename it to `useRandomId`. We want to use it because it is "random", not because it is "fallback"